src/Automation/Automation/Common/AutomationPSClient.cs

```diff
@@ -448,20 +450,143 @@ public void DeleteModule(string resourceGroupName, string automationAccountName,
+            string moduleName)
+        {
+            var createdModule = this.automationManagementClient.PowerShell72Module.CreateOrUpdate(resourceGroupName,
+                automationAccountName,
+                moduleName,
+                new AutomationManagement.Models.ModuleCreateOrUpdateParameters()
+                {
+                    Name = moduleName,
+                    ContentLink = new AutomationManagement.Models.ContentLink()
+                    {
+                        Uri = contentLink.ToString(),
+                        ContentHash = null,
+                        Version = null
+                    },
+                });
+
+            return this.GetPowerShell72Module(resourceGroupName, automationAccountName, moduleName);
+        }
+
+        public Module GetPowerShell72Module(string resourceGroupName, string automationAccountName, string name)
+        {
+            try
+            {
+                var module =
+                    this.automationManagementClient.PowerShell72Module.Get(resourceGroupName, automationAccountName, name);
+                return new Module(resourceGroupName, automationAccountName, module);
+            }
+            catch (ErrorResponseException cloudException)
+            {
+                if (cloudException.Response.StatusCode == System.Net.HttpStatusCode.NotFound)
+                {
+                    throw new ResourceNotFoundException(typeof(Module),
+                        string.Format(CultureInfo.CurrentCulture, Resources.ModuleNotFound, name));
+                }
+
+                throw;
+            }
+        }
+
+        public IEnumerable<Module> ListPowerShell72Modules(string resourceGroupName, string automationAccountName,
+            ref string nextLink)
+        {
+            Rest.Azure.IPage<AutomationManagement.Models.Module> response;
+
+            if (string.IsNullOrEmpty(nextLink))
+            {
+                response = this.automationManagementClient.PowerShell72Module.ListByAutomationAccount(resourceGroupName, automationAccountName);
+            }
+            else
+            {
+                response = this.automationManagementClient.PowerShell72Module.ListByAutomationAccountNext(nextLink);
+            }
+
+            nextLink = response.NextPageLink;
+            return response.Select(c => new Module(resourceGroupName, automationAccountName, c));
+        }
+
+        public Module UpdatePowerShell72Module(string resourceGroupName, string automationAccountName, string name,
+            Uri contentLinkUri, string contentLinkVersion)
+        {
+            try
+            {
+                var moduleModel =
+                this.automationManagementClient.PowerShell72Module.Get(resourceGroupName, automationAccountName, name);
+                if (contentLinkUri != null)
+                {
+                    var updateModule = this.automationManagementClient.Module.CreateOrUpdate(resourceGroupName,
```

Comment:
```suggestion
                    var updateModule = this.automationManagementClient.PowerShell72Module.CreateOrUpdate(resourceGroupName,
```

Reply:
Yes, its should be sorry i missed it. I have Updated the code. Thanks for poiting it out.